src/components/HomePageContent/HowItWorksSection/Steps.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { FunctionComponent } from "react";
-import { StepDetail, StepsProps, ContentType } from "./types";
+import { ContentType, StepDetail, StepsProps } from "./types";
 
 const Step: FunctionComponent<StepDetail> = (props) => {
   const { icon, title, description } = props;
   return (
-    <div className={`flex flex-row flex-none w-6/12 mt-8 justify-center lg:w-auto ${icon ? "" : "lg:ml-6"}`}>
+    <div className={`flex flex-row flex-none w-full xs:w-6/12 lg:w-auto mt-8 justify-center  ${icon ? "" : "lg:ml-6"}`}>
```

Comment:
nitpick (non-blocking): 
```suggestion
    <div className={`flex flex-row flex-none w-full xs:w-1/2 lg:w-auto mt-8 justify-center  ${icon ? "" : "lg:ml-6"}`}>
```

Reply:
ok got it